packages/flutter/lib/src/material/app.dart

```diff
@@ -154,6 +155,9 @@ class MaterialApp extends StatefulWidget {
   /// representative of what will happen in release mode.
   final bool debugShowCheckedModeBanner;
 
+  /// The list of observers for the Navigator created for this app.
```

Comment:
`s/Navigator/[Navigator]/`

Using the `[` braces will make the name a link in the docs web site.